### ISSUE TITLE
[Bug]: Minified javascript assets are not updated

### ISSUE THREAD

**Issue by natted** [pimcore/customer-data-framework#505]
### Expected behavior

In your merged pull request (https://github.com/pimcore/customer-data-framework/pull/504) you fixed cmf.js but did not update the cmf.min.js

### Actual behavior

An old version of cmf.min.js exists. Looking at the file, the minified version in github hasn't been updated for 6 years.

### Steps to reproduce

Is this correct procedure?

Why would you not update the minified version when committing this kind of fix?

If people are supposed to compile the assets again themselves, why have the minified version in the repo at all?

**Comment by aryaantony92** [comment]
@natted Yes, the frontend assets for the admin (src/Resources/public/admin)  is generated by gulp. And at the moment, we have some issues with the build generation. So, we fixed the issue with merge button in this PR https://github.com/pimcore/customer-data-framework/pull/510 which is working in production mode. Thanks!
